packages/odyssey/src/scss/abstracts/_tokens.scss

```diff
@@ -14,7 +14,7 @@
 
 // Typography
 $body-font-family: 'Whyte', '-apple-system',  'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif;
-$mono-font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+$mono-font-family: 'Inconsolata', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
```

Comment:
Is the Inconsolata add official now?

Do we have a distribution plan there? (non-blocking, but should def get ironed out soon if this is official from Brand)

Reply:
It is, currently it is added to the theme itself. There is a comment in the theme fonts.scss file around this